docs/src/api/params.md

```diff
@@ -519,6 +519,24 @@ The default value can be changed by using the [`method: BrowserContext.setDefaul
 Maximum time in milliseconds, defaults to 30 seconds, pass `0` to disable timeout. The default value can be changed by
 using the [`method: AndroidDevice.setDefaultTimeout`] method.
 
+## csharp-select-options-values
```

Comment:
Is it going to be shared by Page and Frame? If not, consider inlining this in csharp.md

Reply:
Yes, likely. But as I'm moving through one by one, I haven't gotten around to that part yet :-).